packages/home-spa/src/service/service.js

```diff
@@ -10,30 +10,46 @@ query HomePageQuery {
   }
 }`;
 
-export function getData() {
+export const getContactUsTeamDetails = `
+query ContactUsQuery{
+  group (cn:"one-portal-devel") {
+    cn
+    name
+    members {
+      name
+      uid
+      _id
+      rhatUUID
+      rhatJobTitle
+    }
+  }
+}
+`;
+
+export function getData(query) {
   const fetchOptions = {
     method: 'post',
     headers: {
-      Authorization: `Bearer ${ window.OpAuthHelper?.jwtToken }`,
+      Authorization: `Bearer ${window.OpAuthHelper?.jwtToken}`,
       'Content-Type': 'application/json',
     },
     body: JSON.stringify({
-      query: getHomeTypeBySPA
-    })
+      query,
+    }),
   };
-  return fetch(process.env.API_URL, fetchOptions)
+  return fetch( process.env.API_URL, fetchOptions )
     .then(res => res.json());
```

Comment:
I know you didn't add this but we should handle HTTP failures as fetch does not handle those more [here](https://www.tjvantoll.com/2015/09/13/fetch-and-errors/#:~:text=Per%20MDN%2C%20the%20fetch(),such%20a%20DNS%20lookup%20failure.)

Reply:
@ghanlohar yah thanks for sharing that ill make these changes with the other one 😄

Reply:
Added an errorhandler part for both data fetch. For now if error occurs it gets consoled like other SPA.